src/frontend/src/handler/create_mv.rs

```diff
@@ -86,15 +86,15 @@ pub fn gen_create_mv_plan(
 
     let definition = context.normalized_sql().to_owned();
 
-    let bound = {
+    let (dependent_views, bound) = {
         let mut binder = Binder::new_for_stream(session);
         let bound = binder.bind_query(query)?;
-        // FIXME: We should record the views into mv's dependent relations to to refine the drop
-        // check and recursive rename of the views.
-        let _views = binder.shared_views().keys().cloned().collect_vec();
-        bound
+        (binder.shared_views(), bound)
```

Comment:
I feel treating views separately here might be confusing and error-prone.. Similar concern with the comment below on `BatchQueryPlanResult`